src/add_compute.jl

```diff
@@ -303,7 +303,7 @@ end
 
 # adds x ^ (p::Real)
 function add_pow!(
-    ls::LoopSet, var::Symbol, x, p::Real, elementbytes::Int, position::Int
+    ls::LoopSet, var::Symbol, @nospecialize(x), p::Real, elementbytes::Int, position::Int
```

Comment:
Will adding this `@nospecialize` have an effect on runtime performance?

Reply:
No. `add_pow!` expands literal powers into a product of squares while the macro expands.

Reply:
I think not but for questions like these I'd defer to the experts.